### PR TITLE
Storyshots: framework option should override package dependency check

### PR DESCRIPTION
**Pull request by ktj**
Issue:
https://github.com/elderfo/react-native-storybook-loader/issues/26
## What I did
In storyshots addon I modified the framework check to make sure framework option overrides dependency check.

## How to test
Reproduction project: https://github.com/ktj/storyshots-rp
```
yarn
yarn test
```
-> tests fail

remove line 8 form package json -> tests work

I've added 3.1.9 example with latest react-native:
https://github.com/ktj/storyshots-rp/tree/319

I added storyshots with this fix as dependency.

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]
I haven't tested, but I think this is a good idea. What about you @tmeasday

**Review by shilman — APPROVED** [review]
LGTM (but didn't test)!

File: examples/react-native-vanilla/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "@storybook/channels": "file:../../lib/channels",
     "@storybook/channel-postmessage": "file:../../lib/channel-postmessage",
     "@storybook/react-native": "file:../../app/react-native",
+    "@storybook/react": "file:../../app/react",
```
Comment (tmeasday) [inline examples/react-native-vanilla/package.json]: Are you just adding this line to prove that the fix works, or does it make sense to have it for the RNV app?

**Review by tmeasday — CHANGES_REQUESTED** [review]
Makes sense to me apart from the package.json change.

File: examples/react-native-vanilla/package.json (hunk 1 above)
Comment (ktj) [inline examples/react-native-vanilla/package.json]: Just to prove it works

Comment (tmeasday) [inline examples/react-native-vanilla/package.json]: Ok can we drop it then? Thanks!

Comment (ndelangen) [inline examples/react-native-vanilla/package.json]: Sounds like this will be super easy to clear up and merge?

**Comment by tmeasday** [comment]
Done, Thanks @ktj !

**Pull request merged**